MiniZn_integration_with_python.py

```diff
@@ -0,0 +1,42 @@
+from minizinc import Instance, Model, Solver
+
+# Load platfroms model from file
+platforms = Model("./platforms.mzn")
```

Comment:
Update the model path in accordance with the existing "master" branch's file setup.

Reply:
Please call appropriate file inside the plaforms.mzn to make the python integration work.